Cargo.toml

```diff
@@ -0,0 +1,45 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+[package]
+name = "datafusion-c"
+description = "DataFusion C API"
+version = "1.0.0"
+homepage = "https://github.com/datafusion-contrib/datafusion-c"
+repository = "https://github.com/datafusion-contrib/datafusion-c"
+readme = "README.md"
+authors = ["Apache Arrow <dev@arrow.apache.org>"]
+license = "Apache-2.0"
+keywords = ["arrow", "c"]
+edition = "2021"
+rust-version = "1.59"
+exclude = [
+  "/cpp/",
```

Comment:
why is this ignored?

Reply:
`/cpp/` is just for reusing `archery` in apache/arrow. It's not needed for packaging.
See also the comment in `cpp/CMakeLists.txt`: https://github.com/datafusion-contrib/datafusion-c/pull/1/files#diff-1bba462ab050e89360fd88110a689e85ee037749cea091a1848ab574381d3795R18

>  # This is just for mimicking apache/arrow repository to use "archery lint --rat".